app/jobs/irs_attempts_events_batch_job.rb

```diff
@@ -42,4 +46,23 @@ def redis_client
   def s3_helper
     @s3_helper ||= JobHelpers::S3Helper.new
   end
+
+  def log_irs_attempts_events_job_metrix_info(result, events, start_time)
+    logger_info_hash(
+      name: 'IRSAttemptsEventJobMetrix',
```

Comment:
Is this associated with [ThreatMetrix](https://github.com/18F/identity-idp/blob/main/app/jobs/threat_metrix_js_verification_job.rb)? If not, could it be changed to `IRSAttemptsEventJob`?

Reply:
No, it is not, I will update this.